assembly error on aarch64

~~~
/tmp/ccDYpY0x.s: Assembler messages:
/tmp/ccDYpY0x.s:302: Error: unknown or missing system register name at operand 2 -- `mrs x0,cntvctss_el0'
/tmp/ccDYpY0x.s:374: Error: unknown or missing system register name at operand 2 -- `mrs xzr,cntvctss_el0'
make[2]: *** [CMakeFiles/arch_timer.dir/build.make:76: CMakeFiles/arch_timer.dir/src/test/arm/arch_timer.c.o] Error 1
~~~

on CPU: 3.0 GHz Ampere® Altra™

~~~
cat /proc/cpuinfo
processor       : 0
BogoMIPS        : 50.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x3
CPU part        : 0xd0c
CPU revision    : 1
~~~

--- a/src/test/arm/arch_timer.c
+++ b/src/test/arm/arch_timer.c
@@ -4,6 +4,9 @@
 #include <sys/prctl.h>
 #include <stdio.h>
 
+// For compatibility with pre-2.38 versions of binutils.
+#define cntvctss_el0 "s3_3_c14_c0_6"
+
 long cntfrq(void) {
   long c;
   __asm__ __volatile__("mrs %0, cntfrq_el0" : "=r"(c));
@@ -19,7 +22,7 @@ long cntvct(void) {
 long cntvctss(void) {
   long c;
   if (getauxval(AT_HWCAP2) & HWCAP2_ECV) {
-    __asm__ __volatile__(".arch armv8.6-a\nmrs %0, cntvctss_el0" : "=r"(c));
+    __asm__ __volatile__(".arch armv8.6-a\nmrs %0, " cntvctss_el0 : "=r"(c));
   } else {
     __asm__ __volatile__("mrs %0, cntvct_el0" : "=r"(c));
   }
@@ -33,7 +36,7 @@ void arch_timer_nops(void) {
   __asm__ __volatile__("mrs xzr, cntfrq_el0");
   __asm__ __volatile__("mrs xzr, cntvct_el0");
   if (getauxval(AT_HWCAP2) & HWCAP2_ECV) {
-    __asm__ __volatile__("mrs xzr, cntvctss_el0");
+    __asm__ __volatile__("mrs xzr, " cntvctss_el0);
   }
 }
 

